R/cnd-abort.R

```diff
@@ -38,24 +38,10 @@
 #'
 #' @section Backtrace:
 #'
-#' Unlike `stop()` and `warning()`, these functions don't include call
-#' information by default. This saves you from typing `call. = FALSE`
-#' and produces cleaner error messages.
-#'
-#' A backtrace is always saved into error objects. You can print a
-#' simplified backtrace of the last error by calling [last_error()]
-#' and a full backtrace with `summary(last_error())`.
-#'
-#' You can also display a backtrace with the error message by setting
-#' the option [`rlang_backtrace_on_error`]. It supports the following
-#' values:
-#'
-#' * `"reminder"`: Invite users to call `rlang::last_error()` to see a
-#'   backtrace.
-#' * `"branch"`: Display a simplified backtrace.
-#' * `"collapse"`: Display a collapsed backtrace tree.
-#' * `"full"`: Display a full backtrace tree.
-#' * `"none"`: Display nothing.
+#' `abort()` always saves a backtrace. You can print a simplified backtrace of
+#' the last error by calling [last_error()] and a full backtrace with
+#' `summary(last_error())`. Control the default behaviour with
```

Comment:
This makes it seem like the error controls the default for `last_error()`. Perhaps:

> You can also print a backtrace immediately when an error is thrown. See [`rlang_backtrace_on_error`].

Reply:
That feels maybe a bit too specific? How about ``Learn how to control what's displayed when an error is thrown with [`rlang_backtrace_on_error`]`` ?

Reply:
yup looks good